sumpy/expansion/loopy.py

```diff
@@ -104,7 +104,8 @@ def make_e2p_loopy_kernel(
             idx = int(insn.assignee.name[len(result.name):])
             insns[i] = lp.Assignment(
                 assignee=result[idx],
-                expression=result[idx] + insn.expression,
+                # FIXME: this should be removed once loopy and pymbolic agree
+                expression=result[idx] + insn.expression,  # type: ignore[arg-type]
```

Comment:
This seems to be because there's some disagreement regarding `ExpressionT` and some scalar types (I think?), so maybe this should wait for loopy changes..

Reply:
> so maybe this should wait for loopy changes..

Agreed. I've got those *almost* ready to go.